### DISCUSSION TITLE
Builders Team Joining Request

### DISCUSSION THREAD

**Discussion by git404x** [ROM-builders/temporary#14288] [Q&A]
Hi, can i join the team ?? 

Github: https://github.com/iTZUDAY2312
Telegram: [@itz_uday_404](https://telegram.me/itz_me_unknown404)

Some Successful builds:
https://cirrus-ci.com/build/5211713418035200
https://cirrus-ci.com/build/5423616098566144
https://cirrus-ci.com/build/5059141222793216
https://cirrus-ci.com/build/4742908174663680

Sources: [miatoll]
[Device tree](https://github.com/iTZUDAY2312/device_xiaomi_miatoll) & [Device Common](https://github.com/iTZUDAY2312/device_xiaomi_sm6250-common)
[Vendor tree](https://github.com/iTZUDAY2312/vendor_xiaomi_miatoll) & [Vendor Common](https://github.com/iTZUDAY2312/vendor_xiaomi_sm6250-common)

i have cherry picked stuffs in device common tree and vendor common as well u can check [CrDroid](https://github.com/iTZUDAY2312/device_xiaomi_sm6250-common/commits/CrDroid) branch

**Comment by Apon77** [comment]
seems all links broken

**Reply by git404x** [reply]
> seems all links broken

sorry i forgot but my git repo pvt , plZ check again all are public now
